int32 visualization_color
---
int32 error